### DISCUSSION TITLE
希望添加自定义弹幕字体和分享当前时间的功能

### DISCUSSION THREAD

**Discussion by Cairl** [the1812/Bilibili-Evolved#1844] [功能建议 / Ideas]
之前旧版播放器有的功能，到了新版播放器就没了。前段时间还可以靠改浏览器字体然后弹幕换成NOTO SANS忍耐一下，最近连Noto Sans都没了，赶紧增加个自定义弹幕字体的功能舅舅孩子吧！

还有希望能增加个分享当前视频时间的功能，这个应该不是很难，但是B站不愿意优化体验

**Comment by Geeyun-JY3** [comment]
第二个需求：#1465

**Comment by the1812** [comment]
弹幕字体见 #807
